### PR TITLE
Remove use of flake8.

### PR DESCRIPTION
**Pull request by tdenewiler**
The flake8 project is marked as abandoned. There are some forks but none that do not cause unsolved issues for our code base.

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -78,11 +78,11 @@ jobs:
...
       - name: Install Hadolint binary (github)
         run: |
-          mkdir hadolint-bin
+          mkdir -p $HOME/.local/bin
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
           curl -sL -o hadolint https://github.com/hadolint/hadolint/releases/download/v2.6.0/hadolint-$(uname -s)-$(uname -m)
           chmod +x hadolint
-          mv hadolint hadolint-bin/.
-          echo "$GITHUB_WORKSPACE/hadolint-bin" >> $GITHUB_PATH
+          mv hadolint $HOME/.local/bin/
```
Comment (xydesa) [inline .github/workflows/tests.yaml:87]: these changes aren't strictly needed right? do you think it's better to use $HOME instead of $GITHUB_WORKSPACE?

Comment (tdenewiler) [reply .github/workflows/tests.yaml:87]: Those changes are not strictly needed. I made them while debugging to come up with the correct overall solution. I found the example from [GitHub docs](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#example-of-adding-a-system-path).

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -93,7 +93,9 @@ jobs:
         run: |
           mypy --ignore-missing-imports --strict src/
 
+      # Do not run on macos until there is a hadolint release for macos (Darwin on arm64 architecture).
       - name: Tox
+        if: matrix.os == 'ubuntu-20.04' || matrix.os == 'ubuntu-22.04' || matrix.os == 'windows-latest'
```
Comment (xydesa) [inline .github/workflows/tests.yaml]: instead of not running tox, why don't we not install hadolint in macos until there's a release for Darwin arm64?

Comment (tdenewiler) [reply .github/workflows/tests.yaml]: [e2551bf](https://github.com/sscpac/statick-tooling/pull/31/commits/e2551bf395cdb15e67f38051e9567523b1194270)

**Review by xydesa — APPROVED** [review]

**Pull request merged**